### PR TITLE
Tx responses refactoring + frontend integration for all txs

### PR DESCRIPTION
**Pull request by andreivasilescu24**

### CODE REVIEW

File: frontend/src/requests/transaction.rs [hunk 1]
```diff
@@ -1,19 +1,58 @@
...
+    PongAll,
+}
+
+pub async fn ping(config: &Config) -> Result<Value, Value> {
     let transaction_url = &config.transaction_url;
     let dest = &config.dest;
-    let full_endpoint = format!("http://{dest}{transaction_url}");
+    let endpoint = format!("http://{dest}{transaction_url}/ping");
+
+    let body = json!({
```
Comment (mihaicalinluca) [inline frontend/src/requests/transaction.rs:16]: place this into an outside form. still not the ideal solution, but good enough until wallet connect

File: microservice/src/routes/helpers.rs [hunk 2]
```diff
@@ -0,0 +1,79 @@
+use core::panic;
+
+use chrono::DateTime;
+use multiversx_sc_snippets::imports::RustBigUint;
+
+pub fn denominate(value: f64) -> String {
```
Comment (mihaicalinluca) [inline microservice/src/routes/helpers.rs:7]: these functions deserve a small unit test

File: microservice/src/routes/helpers.rs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            let result = integer_part + &decimal_part;
+            result.trim_start_matches('0').to_string()
+        }
+    } else {
+        nominated_value.push_str(&"0".repeat(18));
+        nominated_value
+    }
+}
+
+pub fn nominated_str(value: RustBigUint) -> String {
```
Comment (mihaicalinluca) [inline microservice/src/routes/helpers.rs:57]: same as before

**Review by mihaicalinluca — CHANGES_REQUESTED** [review]

**Review by mihaicalinluca — COMMENTED** [review]
use `cargo clippy` to see clippy errors and fix them

File: frontend/src/requests/transaction.rs (hunk 1 above)
Comment (JustEatAnApple) [reply frontend/src/requests/transaction.rs:16]: This is the next feature that we will implement

**Review by mihaicalinluca — APPROVED** [review]

**Pull request merged**